recsys/recsys_tutorial.py

```diff
@@ -58,7 +58,7 @@
 # We look at a sample of the labeled development set.
 # As an example, we want our final recommendations model to be able to predict that a user who has interacted with `book_idxs` (25743, 22318, 7662, 6857, 83, 14495, 30664, ...) would either not read or not like the book with `book_idx` 22764 (first row), while a user who has interacted with `book_idxs` (3880, 18078, 9092, 29933, 1511, 8560, ...) would read and like the book with `book_idx` 3181 (second row).
 
-# %%
+# %% {"tags": ["md-exclude"]}
 df_dev.sample(frac=1, random_state=12).head()
```

Comment:
Fine to exclude this output, but this line needs a comment explaining what's happening regardless.

Reply:
I'm excluding the whole cell, not just the output.